src/main/java/sirius/biz/process/output/TableOutput.java

```diff
@@ -113,4 +114,91 @@ public TableOutput addCellsWithType(List<Cell> cellList, String messageType) {
     public TableOutput addCells(List<Cell> cellList) {
         return addCellsWithType(cellList, null);
     }
+
+    /**
+     * Builder pattern to add {@link Cell} to {@link TableOutput}
+     */
+    public class RowBuilder {
+        private List<Cell> cells;
+
+        private RowBuilder() {
+            cells = new ArrayList<>();
+        }
+
+        /**
+         * Adds a {@link Cell} to a row.
+         *
+         * @param cell the {@link Cell} to add
+         * @return the row builder itself for fluent method calls
+         */
+        public RowBuilder withCell(Cell cell) {
+            this.cells.add(cell);
+            return this;
+        }
+
+        /**
+         * Builds the new row as a collection of cells.
+         *
+         * @return the table output itself for fluent method calls
+         */
+        public TableOutput build() {
+            return addCellsWithType(Collections.unmodifiableList(cells), null);
+        }
+    }
+
+    /**
+     * Initiates a {@link RowBuilder} to receive cells.
+     * <p>
+     * Note that this may only be invoked if <tt>columns</tt> was properly populated when calling the constructor.
+     *
+     * @return the table output itself for fluent method calls
+     */
+    public RowBuilder addRow() {
+        return new RowBuilder();
+    }
+
+    /**
+     * Builder pattern to add columns to a {@link TableOutput}
+     */
+    public static class ColumnBuilder {
```

Comment:
Simply call this "Builder" (?)

Reply:
There are 2 innerclass builders: RowBuilder and ColumnBuilder